articles/contributing/code.md

```diff
@@ -13,13 +13,30 @@ uid: microsoft.quantum.contributing.code
 
-In this guide, we'll detail a bit of what we look for when we review pull requests to help your contribution do the most good.
+In this guide, we'll describe what we look for when we review pull requests to help your contribution do the most good.
 
 ## What We Look For ##
 
-An ideal code contribution builds on the existing work in a Quantum Development Kit repository to fix issues, expand existing features, or to add new features that are within the scope of a repository.
-When we accept a code contribution, it becomes a part of the Quantum Development Kit itself, such that new features will be released, maintained, and developed in the same way as the rest of the Quantum Development Kit.
-Thus, it is helpful when functionality added by a contribution is well-tested and is documented.
+An ideal code contribution builds on the existing work in a Quantum Development Kit repository to fix issues, expand existing features, or to add new features that are within the scope of a repository.  When we accept a code contribution, it becomes a part of the Quantum Development Kit itself, such that new features will be released, maintained, and developed in the same way as the rest of the Quantum Development Kit.  
+
+It is helpful when functionality added by a contribution is well-tested and is documented. We may ask for changes if it requires more documentation or unit tests to help us make use of it, or changes if it differs enough in style from the rest of the Q# libraries that it will make it harder for users to find your feature.  In these cases, we'll try to offer some advice in code reviews about what can be added or changed to make your contribution easier for us to include.  We also may ask for your help in modifying a contribution to better fit into the Quantum Development Kit offering so that we can do the best work we can with it.
+
+Sometimes, a contribution to the quantum programming community is a really good one, but the Quantum Development Kit is not the right place to publish it.  There's a number of reasons why we might not be able to accept a particular contribution.  We try to make sure that the Quantum Development Kit can be used on a variety of platforms, from Linux, to Windows, to MacOS.  We may not accept a contribution that perhaps may not interest a broad enough developer audience.  At other times, we may reject a good contribution simply because we aren't yet ready to maintain and develop it.  In these cases where we cannot accept the contribution, we strongly encourage you to make your own repository --- part of the strength of the Quantum Development Kit is that it's easy to make and distribute your own libraries using GitHub and NuGet.org, the same way that we distribute the canon and chemistry libraries today.  Releasing a feature as a third-party library can make a lot of sense.
```

Comment:
I believe Apple has chosen "macOS" as the capitalization to match "iOS" and "watchOS."

```suggestion
Sometimes, a contribution to the quantum programming community is a really good one, but the Quantum Development Kit is not the right place to publish it.  There's a number of reasons why we might not be able to accept a particular contribution.  We try to make sure that the Quantum Development Kit can be used on a variety of platforms, from Linux, to Windows, and to macOS.  We may not accept a contribution that perhaps may not interest a broad enough developer audience.  At other times, we may reject a good contribution simply because we aren't yet ready to maintain and develop it.  In these cases where we cannot accept the contribution, we strongly encourage you to make your own repository --- part of the strength of the Quantum Development Kit is that it's easy to make and distribute your own libraries using GitHub and NuGet.org, the same way that we distribute the canon and chemistry libraries today.  Releasing a feature as a third-party library can make a lot of sense.
```